guides/delivery/daily-standups.md

```diff
@@ -0,0 +1,39 @@
+# Daily standups
+
+[What are they?](##what-are-they?)
+[Who should join?](##who-should-join?)
+[What does a good standup cover?](##what-does-a-good-standup-cover?)
+[Getting the most out of standups](##getting-the-most-out-of-standups)
+[Other tips](##other-tips)
+
+## What are they?
+A short and focused meeting for the team to check in on progress. They shouldn’t last any longer than 15 minutes; doing them standing up helps keep this focus. 
```

Comment:
I suggest removing "doing them standing up helps keep this focus", or explaining that this was the original thinking behind the name, but it's not a required part of we do them at dxw. (The notion of requiring everyone to stand up is ableist, and you've already included guidance on how to achieve the goal of keeping focused and timely.)

Reply:
Thanks @CristinaRO - I've taken that sentence out 😀.